src/main/java/com/googlecode/lanterna/terminal/DefaultTerminalFactory.java

```diff
@@ -408,7 +408,12 @@ private Terminal createWindowsTerminal() throws IOException {
             return (Terminal)constructor.newInstance(inputStream, outputStream, charset, UnixLikeTTYTerminal.CtrlCBehaviour.CTRL_C_KILLS_APPLICATION);
         }
         catch(Exception ignore) {
-            return createCygwinTerminal(outputStream, inputStream, charset);
+            try {
+                return createCygwinTerminal(outputStream, inputStream, charset);
+            } catch(IOException e) {
+                throw new IOException("To start java on Windows, use javaw!\n"
+                        + "(see https://github.com/mabe02/lanterna/issues/335 )", e);
```

Comment:
Maybe use System.lineSeparator() instead of "\n" for better Windows support?

Reply:
Yes, of course, I forgot about it...